databases/week3/funcionalityQueries.sql

```diff
@@ -0,0 +1,37 @@
+-- Get Meals that has a Price smaller than a specific Price fx 90
+SELECT * FROM Meal 
+WHERE Price < 90; 
+
+-- Get Meals that still has available reservations
+SELECT * FROM Meal 
+WHERE Max_reservations > 0; 
+
```

Comment:
You need to somehow figure out the number of reservations contra max possible reservations and see if there still are vacancies.